src/librustc_typeck/diagnostics.rs

```diff
@@ -4146,8 +4102,6 @@ register_diagnostics! {
 //  E0068,
 //  E0085,
 //  E0086,
-    E0103, // @GuillaumeGomez: I was unable to get this error, try your best!
-    E0104,
```

Comment:
The general rule of thumb is "don't reuse error codes".  It's okay to deprecate and just not output that code anymore, but reusing tends to mess with search results.